smithy-aws-protocol-tests/model/rpcV2/empty-input-output.smithy

```diff
@@ -83,29 +83,29 @@ use smithy.test#httpResponseTests
         code: 200,
     },
     {
-        id: "no_output_client_allows_accept",
+        id: "NoOutputClientAllowsEmptyCbor",
         protocol: rpcv2Cbor,
-        documentation: "Servers should allow the accept header to be set to the default content-type.",
+        documentation: "Client should accept CBOR empty struct if no output",
         headers: {
             "smithy-protocol": "rpc-v2-cbor",
-            "Accept": "application/cbor",
             "Content-Type": "application/cbor"
         }
-        body: "",
         code: 200,
+        bodyMediaType: "application/cbor",
+        body: "v/8=",
         appliesTo: "client",
     },
     {
-        id: "no_input_client_allows_empty_cbor",
+        id: "NoOutputClientAllowsEmptyBody",
         protocol: rpcv2Cbor,
-        documentation: "Client should accept CBOR empty struct if no output",
+        documentation: "Client should accept empty body if no output",
```

Comment:
The difference between this test and `no_output` is the `Content-Type` header, let's call that out here.